File: lab1 stack, queue, deque/D.py
import sys
a, b, c = [], [], []

for s in sys.stdin:
    if s[0] == '9':
        a.append(s[2:])
    elif s[:2] == '10':
        b.append(s[3:])
    elif s[:2] == '11':
        c.append(s[3:])

while a:
    print('9 ' + a.pop(0), end='')
while b:
    print('10 ' + b.pop(0), end='')
while c:
    print('11 ' + c.pop(0), end='')

File: lab1 stack, queue, deque/D.cpp
#include <iostream>
#include <queue>

using namespace std;

int main(){
    queue<pair<int, string> > a, b, c;
    string s;
    int i;

    while (cin >> i){
        cin >> s;
        if (i == 9){
        a.push(make_pair(i, s));
        }else if (i == 10){
        b.push(make_pair(i, s));
        }else if (i == 11)
        c.push(make_pair(i, s));
    }

    while (!a.empty()){
        cout << a.front().first << ' ' << a.front().second << endl;
        a.pop();
    }
    while (!b.empty()){
        cout << b.front().first << ' ' << b.front().second << endl;
        b.pop();
    }
    while (!c.empty()){
        cout << c.front().first << ' ' << c.front().second << endl;
        c.pop();
    }
    return 0;
}